test/models/order_test.rb

```diff
@@ -10,4 +14,17 @@ class OrderTest < ActiveSupport::TestCase
 
     assert order.save
   end
+
+  test "can filter orders by pending status" do
+    pending_orders = [Order.create(stock: Stock.first, shares: 5, status: :pending, user: @student)]
+    Order.create(stock: Stock.first, shares: 5, status: :completed, user: @student)
+
+    assert_equal pending_orders, Order.pending
+  end
+
+  test "calculates purchase cost" do
+    stock = Stock.create(ticker: "EVG", price: 10.00)
```

Comment:
There should already be a stock, defined in the fixtures folder, that exists when this test is run that can use instead of creating a new one

Reply:
Updated to use fixture. Got it, I usually use FactoryBot for these types of tests

Reply:
I like FactoryBot. We may eventually want it on this project. I like the idea of using the fixtures for now until we have a compelling reason to switch